dev/2c67e4d5-e2b1-4d65-8676-bbd15665b5bd/device.lp

```diff
@@ -0,0 +1,36 @@
+(librepcb_device 2c67e4d5-e2b1-4d65-8676-bbd15665b5bd
+ (name "TCA9548A I2C-MUX")
+ (description "")
```

Comment:
Same as above, adjust name and description. General question how do we manage the duplication of such names and description? Or where is the description most important for the end user?

Reply:
If you are creating a **component** you search for **symbols** and get the **symbols description**.

If creating a **device** you search for a **package and component** and get their description.

If you are adding **devices** to your schematic you get the **devices description**.

In my opinion, for the TCA9548A the description can be the same for the component, package, device.

Reply:
Full ACK @mwit-a :smiley: 

Basically, symbol/package descriptions are only relevant for library designers, while component/device descriptions are relevant for schematic/board designers (which is more important, so generally those should be documented better than symbols and packages).